riot/riot.py

```diff
@@ -109,6 +109,16 @@ def venv_path(self) -> str:
         version = self.version().replace(".", "")
         return f".riot/venv_py{version}"
 
+    def site_packages_path(self) -> str:
```

Comment:
can this be tested?

Reply:
ok, I tried, I don't think I _love_ the way the test is setup, would love feedback on how to better approach.